src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -194,6 +194,7 @@ static String getResource(String key)
 				{"R_packetSizePropertyDescription", "The network packet size used to communicate with SQL Server."},
 				{"R_encryptPropertyDescription", "Determines if Secure Sockets Layer (SSL) encryption should be used between the client and the server."},
 				{"R_trustServerCertificatePropertyDescription", "Determines if the driver should validate the SQL Server Secure Sockets Layer (SSL) certificate."},
+				{"R_trustStoreTypePropertyDescription", "Type of trust store type like JKS / PKCS12 or nay FIPS implementation KeyStore Type. etc."},
```

Comment:
Are these the allowable types or are there others? http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#KeyStore

Perhaps we should simply say 'Type of trust store, such as JKS or PKCS12.'